test/deploy-eks-cluster.sh

```diff
@@ -16,6 +16,14 @@ if [[ -z "${ECR_REPOSITORY}" ]]; then
 fi
 
 function deleteCluster() {
+  NODE_GROUP=$(eksctl get nodegroup --cluster=${TEST_CLUSTER_NAME} | sed -n 4p | awk '{ print $2 }')
+  if [[  ! -z "${NODE_GROUP}" ]]; then
+    eksctl delete nodegroup --cluster=${TEST_CLUSTER_NAME} --name=${NODE_GROUP}
```

Comment:
Will this handle cleaning up the volumes attached to these nodegroup instances (EBS)?

Reply:
Yes it will cleanup the volumes.